Sources/PackageGraph/PackageGraphRoot.swift

```diff
@@ -16,17 +16,14 @@ import SourceControl
 
 /// Represents the input to the package graph root.
 public struct PackageGraphRootInput {
-
-    public typealias PackageDependency = PackageGraphRoot.PackageDependency
```

Comment:
Nice!  Although the type aliases have made things more ergonomic, they have always made the code a bit harder to read, I think.

Reply:
Yeah, I've also found type aliases to be a mixed bag. Unless they're part of a protocol associated type requirement, I've only occasionally found the ergonomics to justify the trade-offs in comprehension.